sources/packages/backend/apps/api/src/services/education-program/education-program.service.ts

```diff
@@ -119,13 +122,25 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
         );
     }
 
+    const isSABCCodeDuplicate = await this.hasExistingProgramWithSameSABCCode(
```

Comment:
If `educationProgram.sabcCode` is "false" we can skip the call to `hasExistingProgramWithSameSABCCode` and it also does not need to be part of the check on line 130, unless I am missing something 😉

Reply:
Yes, the query works because it is `null` in SQL and null comparisons never return true but we can avoid doing the query to save resources here for sure. Thanks!